test/helpers/search_hepler.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class SearchHelperTest < ActionView::TestCase
+  test 'return whether talk' do
+    user = users(:kimura)
+    assert talk?(user)
+
+    user = users(:taikai3)
+    assert_not talk?(user)
```

Comment:
複数回使う必要がないのであれば代入せずこのようにするとスマートかもです〜

```suggestion
    assert talk?(users(:kimura))
    assert_not talk?(users(:taikai3))
```

Reply:
ご指摘の通り冗長な変数宣言だと思いましたので修正いたしました。